### ISSUE TITLE
Only possible to modify/delete event from a daily view if day is at the top of the view

### ISSUE THREAD

**Issue by jose1711** [anufrievroman/calcure#85]
I would say it's not very intuitive that in the daily view the actions are only effective for the day at the top. I think it would be better if it's possible to delete/move/rename any event visible in a daily view.

**Comment by anufrievroman** [comment]
Yes, I noticed that inconvenience as well, but this requires quite a substantial change in the view system it seems, I'll take a look.

**Comment by anufrievroman** [comment]
Took a couple of years, but finally fixed :D
